taskwarrior-web.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |s|
   s.add_dependency('activesupport', '~> 3')
   s.add_dependency('sinatra-simple-navigation')
   s.add_dependency('rack-flash3')
-  s.add_dependency('json', '~> 1.7.7')
+  s.add_dependency('json', '~> 1.8')
```

Comment:
Is this likely to cause backwards compatibility issues? This is required for newer versions of Ruby, but I'm not sure what effect this will have on older versions.